docs/modules/ROOT/pages/user-guide/advanced-soap-server-topics.adoc

```diff
@@ -0,0 +1,63 @@
+= Advanced SOAP server topics
+
+== Running behind a reverse proxy
+
+SOAP requests aimed towards services running on Quarkus can be routed through proxies that generate additional headers
+(e.g. `X-Forwarded-Host`) to keep information from the client-facing side of the proxy servers that is altered or lost when they are involved.
+In those scenarios, Quarkus can be configured to automatically update information like protocol, host, port and URI reflecting the values in these headers.
+
+TIP: Refer to https://quarkus.io/guides/http-reference[Quarkus HTTP reference] for more details.
+
+{quarkus-cxf-project-name} support for various `X-Forwarded` headers works in line with Quarkus configuration.
+
+IMPORTANT: Activating this feature leaves the server exposed to several security issues (i.e. information spoofing). Consider activate it only when running behind a reverse proxy.
```

Comment:
Consider **_activating_** it only when .... (looks like an issue with the Quarkus docs as well).

Reply:
Let's make our copy better than their original :D